portal-web/test/functional/com/liferay/portalweb/dependencies/test-theme.war/WEB-INF/liferay-look-and-feel.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
```

Comment:
Nice, I was wondering how to deploy something through deploy folder. This folder is about 10MB, on the heavy side but I think we can make a case for this if we reuse this dependency file for our Themes testing.

Reply:
Thanks for reviewing this! I didn't consider about the size of the folder. I think we still have some new tests that need to deploy Custom Theme, but we can do that through App Manager. Can we just deploy theme through deploy folder in one test, and through App Manager in other tests? Then we can keep the war file, but need to find another way to deploy through deploy folder.

Reply:
I asked in poshi slack channel if there's a way to deploy zipped files in deploy folder.

Reply:
Based on the response, let's go with the deploy route. I actually prefer if we use the deploy folder on all of our tests and avoid the App Manager or just use it in one test. The App Manager is outside of FI team's control so I want to decouple as many dependencies from other teams as possible that may introduce variance or risk. Deploy folder with ArchiveUtil.archive seems more stable to me.